File: PSet1/mario_less.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do 
    {
        n = get_int("Rows: ");
    }
    while (n <= 0 || n >= 9);
    for (int i = 1; i <= n; i++)
    {
        for(int j = 1; j <= (n-i); j++)
        {
            printf(" ");
        }
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: PSet6/mario_less.py
from cs50 import get_int

# compiles all codes
def main():
    n = get_integer()
    print_blocks(n)
    print(end="")


# Gets integar from user command prompt
def get_integer():
    while True:
        n = get_int("Height: ")
        if n > 0 and n < 9:
            break
    return n

# print blocks depending on input
def print_blocks(n):
    for i in range(n):
        print(" " * (n-i-1) + "#" * (i+1))


main()
